### ISSUE TITLE
engine-q is inserting CRs where they shouldn't be

### ISSUE THREAD

**Issue by fdncred** [nushell/engine-q#968]
I assume this is like what happened in nushell that every xxx number of characters a CR is inserted. I don't remember why but it seems to be happening again in engine-q.

row 47 and 48 should be 1 row - I believe this is 978 bytes into the string, if I counted correctly.
<img width="671" alt="Screen Shot 2022-02-06 at 9 09 36 PM" src="https://user-images.githubusercontent.com/343840/152718963-0333adce-6a05-44c4-b1bc-7052e67909c5.png">

the string is generated in my config.nu like this `let-env LS_COLORS = (vivid generate molokai | str trim)`

my config also has this to parse the LS_COLORS env var
```
    env_conversions: {
        "PATH": {
            from_string: { |s| $s | split row (char esep) }
            to_string: { |v| $v | str collect (char esep) }
        }
        "LS_COLORS": {
            from_string: { |s| $s | split row (char esep) }
            to_string: { |v| $v | str collect (char esep) }
        }
        "DYLD_FALLBACK_LIBRARY_PATH": {
            from_string: { |s| $s | split row (char esep) }
            to_string: { |v| $v | str collect (char esep) }
        }
    }
```

**Comment by jntrnr** [comment]
I suspect that's not a newline, but maybe two frames of input. If I'm right, you can `str collect` before `str trim`.

This feels like another vote that `str trim` should auto-collect its input.

**Comment by jntrnr** [comment]
I put #969 together to test my hunch. Can you give it a try to see if it fixes your issue?

**Comment by jntrnr** [comment]
fixed with #969
